pkg/cmd/step/create/helmfile/create_helmfile.go

```diff
@@ -82,68 +85,109 @@ func NewCmdCreateHelmfile(commonOpts *opts.CommonOptions) *cobra.Command {
 
 // Run implements the command
 func (o *CreateHelmfileOptions) Run() error {
-
-	apps, err := config.LoadApplicationsConfig(o.dir)
+	apps, _, err := config.LoadAppConfig(o.dir)
```

Comment:
I realise that _ CreateHelmfileOptions_ already exists now, but it would have been nice to implement this new Helmfile support outside of this Options framework. One thing we need to get away from is this whole Option hierarchy thing.